examples/graphbolt/rgcn/download.py

```diff
@@ -0,0 +1,670 @@
+import argparse, hashlib, os, shutil, tarfile, yaml
+import subprocess
+import urllib.request as ur
+
+import dgl.graphbolt as gb
+import numpy as np
+from tqdm import tqdm
+
+GBFACTOR = 1 << 30
+
+
+def build_yaml_helper(path, dataset_size, in_memory=True):
+    """The stirng to build the yaml file. (Still need modification)"""
+    data = {
+        "graph": {
+            "nodes": [
+                {"num": num_nodes[dataset_size]["paper"], "type": "paper"},
+                {"num": num_nodes[dataset_size]["author"], "type": "author"},
+                {
+                    "num": num_nodes[dataset_size]["institute"],
+                    "type": "institution",
+                },
+                {
+                    "num": num_nodes[dataset_size]["fos"],
+                    "type": "field_of_study",
+                },
+            ],
+            "edges": [
```

Comment:
Let's modify edges as done in this file to add the reverse direction as well:
https://github.com/mlcommons/training/blob/cdd928d4596c142c15a7d86b2eeadbac718c8da2/graph_neural_network/compress_graph.py#L48-L56

Reply:
so we store another version copy of the reverse edge in the edges dir?

Reply:
Yes, see how they did here: https://github.com/mlcommons/training/blob/cdd928d4596c142c15a7d86b2eeadbac718c8da2/graph_neural_network/compress_graph.py#L73-L84

Reply:
https://mlcommons.org/2024/06/gnn-for-mlperf-training-v4/ here they say they add reverse edges for better accuracy.

Reply:
https://github.com/mlcommons/training/blob/cdd928d4596c142c15a7d86b2eeadbac718c8da2/graph_neural_network/compress_graph.py#L47